scripts/cipublish

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${ECHOLOCATOR_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+         "Usage: $(basename "$0")
+
+Publish deployment artifacts in CI.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]
+then
+    if [ "${1:-}" = "--help" ]
+    then
+        usage
+    else
+        TAUI_CONFIG_BUCKET="s3://echo-locator-${ENV}-config-us-east-1"
+        TAUI_CONFIG_DIR="configurations/${ENV}"
+        TAUI_SITE_BUCKET="s3://echo-locator-${ENV}-site-us-east-1"
+
+        # Pull down variables for this environment
+        aws s3 cp "${TAUI_CONFIG_BUCKET}/taui/env.yml" \
+                  "taui/${TAUI_CONFIG_DIR}/env.yml"
```

Comment:
`env.yml` is the only Mastarm configuration file that contains secrets. Because of this, I figured the safest way to handle it was to keep versions for each environment in the S3 remote state bucket and pull it down during the CI build.

I think it would make sense to have one of these for development and do a similar thing in an `update` script. I didn't do that in this PR since I didn't really plan to invest too much time in the dev environment.

Reply:
Could we add a local `env.yml` example file and instructions to the `README` to copy it over and set secrets for development setup here instead?

Reply:
Yup, that works! I added some very basic instructions to the README. You may need to adjust these instructions depending on what winds up being configured in `env.yml`.